### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by tarikuzuma**
Created installation guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# README
```
Comment (fofajardo) [inline README.md]: nit: use full game name here instead of just readme.

**Review by fofajardo — CHANGES_REQUESTED** [review]
Please see review comments. Otherwise, LGTM!

**Comment by tarikuzuma** [comment]
I changed readme.md to Dikta.md, can you check the commit?

**Review by fofajardo — APPROVED** [review]
LGTM! Merging.

**Pull request merged**